packages/web/components/config/SessionConfigPanel.tsx

```
@@ -74,20 +62,32 @@ const DEFAULT_CONFIG: SessionConfiguration = {
-  sessions,
-  selectedSession,
-  providers,
-  onSessionCreate,
-  onSessionSelect,
-  onAgentCreate,
-  onAgentSelect,
-  onAgentUpdate,
-  onSessionUpdate,
-  loading = false,
-}: SessionConfigPanelProps) {
+export function SessionConfigPanel({}: SessionConfigPanelProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)